src/lib/mana.ts

```diff
@@ -1,5 +1,23 @@
+import { ChainId } from '@dcl/schemas'
+import { Authorization, AuthorizationType } from 'decentraland-dapps/dist/modules/authorization/types'
+import { ContractName, getContract } from 'decentraland-transactions'
+
 const MANA_SYMBOL = '⏣'
 
 export function addSymbol(num: string | number) {
   return num > 0 ? `${MANA_SYMBOL} ${num.toString()}` : ''
 }
+
+export function buildManaAuthorization(address: string, chainId: ChainId, contractName: ContractName): Authorization {
```

Comment:
Maybe we can test this method?

Reply:
Oh, I forgot to add tests to it! Added them.